src/Constants.ts

```diff
@@ -6,4 +6,10 @@ export const CATALOG_FILE = 'Catalog.json';
 export const CUSTOM_PATHS_FILE = 'CustomPaths.json';
 export const CONFIG_FILE = 'Config.json';
 
-export const TYPESCRIPT_RULE_PREFIX = '@typescript';
\ No newline at end of file
+export const TYPESCRIPT_RULE_PREFIX = '@typescript';
+
+export const LANGUAGES = {
```

Comment:
@rmohan20 based on the ENGINE enum in https://github.com/forcedotcom/sfdx-scanner/pull/121 I will convert this to an enum. I'll also make it singular `LANGUAGE`

Reply:
Awesome :)